shell/platform/darwin/ios/framework/Source/accessibility_bridge.h

```diff
@@ -129,6 +131,10 @@ class AccessibilityBridge final {
 
   fml::WeakPtr<AccessibilityBridge> GetWeakPtr();
 
+  FlutterPlatformViewsController* flutter_platform_views_controller() const {
```

Comment:
... `GetPlatformViewsController() const` for consistency with the rest of the engine.

Reply:
done